docs/storefront/headless/routes.mdx

```diff
@@ -0,0 +1,269 @@
+# Routes with the GraphQL Storefront API
+
+The BigCommerce GraphQL Storefront API allows you to retrieve data at the destination of a route (URL) for each storefront channel using the `route` node. This option makes it easier to build headless storefront applications, removing the need to track entity ID and type when accessing data for entities such as Products, Categories, Pages, Blogs, and more.
+
+The only required input when using the `route` node is the destination URL. The response will be an entity at that URL, a redirect to a new location, or nothing if the URL does not relate to an entity or redirect.
```

Comment:
Lets mention the input parameter here...I'm thinking a 2nd sentence in the existing paragraph like:

> The only required input when using the `route` node is the destination URL. Input is provided with the `path` param on the `route` node. The response will be an entity at that URL...

Reply:
Thank you @bc-jz  I changed line 5 as follows:

The destination URL is the only required input when using the route node. You will provide an input with the path parameter on the route node. The response will be an entity at that URL, a redirect to a new location, or nothing if the URL does not relate to an entity or redirect.